### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ramkrishna2910**
This PR updates the feedback on Readme and contribute guides.

### CODE REVIEW

File: contribute.md [hunk 1]
```diff
@@ -27,50 +27,50 @@ There are many ways you can contribute to the ONNX Model Zoo:
...
+The source file must include the following labels at the top of the file:
+    
+    - Author - Name of the organization or the creator of the model
+    - Task - For example, Computer Vision, Natural Language Processing (NLP), Audio Processing, MultiModal, Generative AI ,Graph Machine Learning ...
+    - License: Submissions should be under one of the following licenses: Apache-2.0, BSD-3, or MIT.
+    
+    For the correct format of these labels, please refer to the example here (todo: add link).
 
-    By following these guidelines, you help us maintain a high standard for the models in the ONNX Model Zoo, making it a valuable resource for everyone. We appreciate your contributions and look forward to including your models in our collection.
+1. **Dependencies**: Add any additional requirements run your model to `onnx\models\toolchain\model\requirements.txt`(todo: insert link) file.
```
Comment (ramkrishna2910) [inline contribute.md:59]: Lets just stick to one requirements file for now until we go to a per corpus requirements.

Comment (jeremyfowers) [inline contribute.md:59]: Your numbering starts over from 1. here in the rendered doc

File: contribute.md [hunk 2]
```diff
@@ -27,50 +27,50 @@ There are many ways you can contribute to the ONNX Model Zoo:
...
-    1. **Dependencies**: Include a requirements.txt file detailing all the necessary dependencies to run your model.
+To contribute a model to the ONNX Model Zoo, we ask that you adhere to the following guidelines to ensure consistency and quality:
 
-    1. **Documentation**: While optional, we highly encourage you to provide a README file with:
-        - A detailed description of the model.
-        - Instructions for use, including setup and execution.
+1. **Model Uniqueness**: Before submission, please search the Model Zoo to confirm that your model or an equivalent model is not already hosted.
 
-    1. **Large Files**: If your model includes large weight files, manage them with Git LFS.
+1. **Model Source**: You should provide the source file for the model (.py format).
```
Comment (jeremyfowers) [inline contribute.md:47]: to `toolchain/models`?

File: contribute.md [hunk 3]
```diff
@@ -27,50 +27,50 @@ There are many ways you can contribute to the ONNX Model Zoo:
...
 
-    1. **Model Building**: Use TurnkeyML's `--build-only` (todo: add link) option to generate the ONNX model. This ensures that the model conforms to standards compatible with the Model Zoo's infrastructure.
+1. **Source File**: Place the source file under `onnx\models\toolchain\models` and place the onnx file under the appropriate task category in the repo root. All pretrained models should include valid weights.
```
Comment (jeremyfowers) [inline contribute.md]: These two items seem redundant

File: contribute.md [hunk 4]
```diff
@@ -27,50 +27,50 @@ There are many ways you can contribute to the ONNX Model Zoo:
...
-    ## Toolchain Contribution Guidelines
+1. **Documentation**: While optional, we highly encourage you to provide a README file with:
+    - A detailed description of the model.
+    - Instructions for use, including setup and execution.
 
-    For contributing to the toolchain under `onnx\models\toolchain` plese refer to the contribution guidelines here. (todo: add link)
+1. **Large Files**: If your model includes large weight files, manage them with Git LFS.
 
-5. **Commit Messages**: Use clear and descriptive commit messages. This helps to understand the purpose of your changes and speed up the review process.
+1. **Model Building**: Use TurnkeyML's `--build-only` (todo: add link) option to generate the ONNX model. This ensures that the model conforms to standards compatible with the Model Zoo's infrastructure.
```
Comment (jeremyfowers) [inline contribute.md]:
```suggestion
1. **ONNX file creation**: Use TurnkeyML's `--build-only` (todo: add link) option to generate the ONNX model. This ensures that the model conforms to standards compatible with the Model Zoo's infrastructure.
```

suggest being more specific

**Review by jeremyfowers — APPROVED** [review]
Looking good! I suspect we will refine over time as people actually try to contribute ;)

File: contribute.md (hunk 3 above)
Comment (ramkrishna2910) [reply contribute.md]: Combined

File: contribute.md (hunk 1 above)
Comment (ramkrishna2910) [reply contribute.md:59]: Fixed

**Pull request merged**